tests/test_device.py

```diff
@@ -352,12 +352,6 @@ def test_socket_state_on():
     assert socket.state is True
 
 
-def test_set_state_none(device):
```

Comment:
Should we keep this? We can't send `None` to set_state as it's typed for `int` now.

Reply:
I don't think we need this test. Although during run time we could still pass None, code wise it would fail our static typing checks.

Reply:
OK, then I'll leave it deleted